net/qobuz-dl/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                qobuz-dl
+version             0.9.9.5
+revision            0
+
+categories          net
+platforms           darwin
+supported_archs     noarch
+license             GPL-3
+maintainers         {@catap korins.ky:kirill} openmaintainer
+
+description         The complete Lossless and Hi-Res music downloader for Qobuz
+long_description    ${description}
+
+homepage            https://github.com/vitiko98/Qobuz-DL
+
+checksums           rmd160  abdfee278f5ee9271f350ec589536902b63fad67 \
+                    sha256  19d1f81be8d123da2304702d6f8fed36b963652515a5548b9c108ed65ffd2ce5 \
+                    size    35811
+
+python.default_version 310
+
+depends_build-append \
+                    port:py${python.version}-setuptools
+
+depends_lib-append \
+                    port:py${python.version}-beautifulsoup4 \
+                    port:py${python.version}-colorama \
+                    port:py${python.version}-mutagen \
+                    port:py${python.version}-pathvalidate \
+                    port:py${python.version}-pick \
+                    port:py${python.version}-requests \
+                    port:py${python.version}-tqdm
+
+post-extract {
+    # 0.9.9.5 had missed requirements.txt
+    system -W ${worksrcpath} "touch requirements.txt"
+}
+
+livecheck.type  none
```

Comment:
this is probably not what you intended - why would you not have the `livecheck` enabled here?

Reply:
Good catch, and probably just a minor oversight. Kirill?

Reply:
Yes, defently a typo.